src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -22,7 +22,9 @@
             <argument /> <!-- Format -->
             <argument type="collection" /> <!-- Context -->
         </service>
-
+        <service id="Symfony\Component\Messenger\Transport\Serialization\SerializerInterface"
+                 alias="messenger.transport.symfony_serializer">
+        </service>
```

Comment:
We need a blank line here (for consistency and readability).

Reply:
Done